.github/workflows/ci.yml

```diff
@@ -380,32 +379,18 @@ jobs:
 
       # Deploy dev
       - name: Deploy k8s to cluster
-        uses: wahyd4/kubectl-helm-action@master
-        env:
-          KUBE_CONFIG_DATA: ${{ secrets.KUBECONFIG_NEOTOOLSUITE_DEV }}
-        with:
-          args: |
-            helm upgrade --install matrix-neoboard ./charts/matrix-neoboard-widget -f ./valhalla/cluster/neotoolsuite-dev/neoboard/values-neotoolsuite-dev.yaml -n matrix --set image.tag="argo-${{ needs.build.outputs.ARGOCD_SHA }}"
+        run: |
+          helm upgrade --install matrix-neoboard ./charts/matrix-neoboard-widget -f ./valhalla/cluster/neotoolsuite-dev/neoboard/values-neotoolsuite-dev.yaml -n matrix --set image.tag="argo-${{ needs.build.outputs.ARGOCD_SHA }}"
+
       - name: Restart rollout
-        uses: wahyd4/kubectl-helm-action@master
-        env:
-          KUBE_CONFIG_DATA: ${{ secrets.KUBECONFIG_NEOTOOLSUITE_DEV }}
-        with:
-          args: |
-            kubectl rollout restart deployment matrix-neoboard -n matrix
-            #
+        run: |
+          kubectl rollout restart deployment matrix-neoboard -n matrix
```

Comment:
can/should we remove these rollout steps entirely? it sounds to me like our chart is broken when a helm upgrade is not enought?

Reply:
seems the same to me. I suggest for now keeping it until we were able to properly review and fix the charts. We know there may be bugs...